WTWSMS/common/landed_titles/WtWSMS_holy_orders.txt

```diff
@@ -226,4 +226,55 @@ d_melchisedechianism_holy_order = {
 	monthly_income = 2
 
 	mercenary_type = zealots_composition
-}	
\ No newline at end of file
+}
+
+c_priscillianism_holy_order = {
```

Comment:
What unforeseen issues? Why must it be a duchy? Why can't the Priscillian religious head be king-tier when they are definitively Catholicand papal in structure?